temporalcli/commandsgen/commands.yml

```diff
@@ -605,6 +605,179 @@ commands:
         description: Reason for terminating the batch job.
         required: true
 
+  - name: temporal worker
+    summary: Read or update Worker state
+    description: |
+      Modify or read state associated with a Worker, for example,
+      using Worker Deployments commands:
+
+      ```
+      temporal worker deployment
+      ```
```

Comment:
Let's also mark this as experimental; no need to stabilize something unnecessarily. :)